src/pages/distroSettings/tabs/ProviderTab/getFormSchema.ts

```diff
@@ -141,7 +136,39 @@ export const getFormSchema = ({
                         enum: [r],
                       })),
                     },
-                    ...ec2FleetProviderSettings,
+                    ...ec2FleetProviderSettings.schema,
+                  },
+                },
+              },
+            },
+          },
+          {
+            properties: {
+              provider: {
+                properties: {
+                  providerName: {
+                    enum: [Provider.Ec2OnDemand],
+                  },
+                },
+              },
+              ec2OnDemandProviderSettings: {
```

Comment:
Not sure how fixable this is (or if it needs fixing) but when I switch to EC2 on demand from another provider the card title for the first region is blank

<img width="656" alt="image" src="https://github.com/evergreen-ci/spruce/assets/9298431/f2483db6-0bb8-4346-8ba9-b7cd22d58e17">

Reply:
no problemo, just gotta tweak the transformer :)